### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by ghost**
- [DEMO LINK](https://perronogrande.github.io/layout_catalog_hovers/)

### CODE REVIEW

**Review by caelum29 — APPROVED** [review]
👍

**Review by aliceblunt8 — CHANGES_REQUESTED** [review]

**Review by herkoss — CHANGES_REQUESTED** [review]
Smoothly (during 300ms) change navigation link text color to #00acdc

File: src/styles/_header.scss [hunk 1]
```diff
@@ -37,5 +37,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition: transform 300ms, color 300ms;
```
Comment (herkoss) [inline src/styles/_header.scss:40]: Why do you need a transform property here?

**Review by herkoss — APPROVED** [review]
LGTM👌

**Pull request closed without merging**